.markdownlint/customRule.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
           "lineNumber": heading.lineNumber,
           "detail": "First heading should be '# Change Log'.",
         });
-      } else if (heading.tag === "h2" && !/^## \d\.\d{1,9}\.\d{1,3}-?[A-Za-z0-9]{0,} - \d{4}-\d{2}-\d{2}$/.test(heading.line)) {
+      } else if (heading.tag === "h2" && !/^## \d\.\d{1,9}\.\d{1,3}(-[.A-Za-z0-9]+)? - \d{4}-\d{2}-\d{2}$/.test(heading.line)) {
```

Comment:
Unrelated but fixing to more fully support semver pre-release pattern (so we can use it in this PR)